src/MatLang.jl

```diff
@@ -1,23 +1,167 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, zerosM, onesM, randM, eyeM, diagM
+
+# Julia functions with M suffix
+# export
+
+################################################################
+################################################################
+# Command Window (REPL console):
+################################################################
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
 end
-################################################################
-
-
-################################################################
 # Conditional REQUIRE - https://github.com/MikeInnes/Requires.jl
 function __init__()
     @require Juno="e5e0dc1b-0480-54bc-9374-aad01c23163d" clcM() = Juno.clearconsole();
 end
+################################################################
+
+
+################################################################
+################################################################
+# Matrices and Arrays:
+################################################################
+################################################################
+"""
+    zerosM(sizeAsArray) # non-efficient Matlab way
+
+# Example
+mZeros=zerosM(2,2); # same as zeros(2,2)
+
+mZeros2=zerosM(Int32,2,2); # same as zeros(Int32,2,2)
+
+mZeros3=zerosM((2,2)); # giving size as Tuple
+
+mZeros4=zerosM(Int32,(2,2)); # giving size as Tuple and also the Int32 type
+
+mZeros5=zerosM([2,2]); # giving size as an Array, non-efficient Matlab way. Array should be Integer otherwise you will get errors.
+
+mZeros6=zerosM(Int32,[2,2]); # giving size as Array, non-efficient Matlab way
+"""
+zerosM(args...) = zeros(args...)
+zerosM(a::Array)=zeros(Tuple(a))
```

Comment:
Please do not use `Array` as the internal implementation, it's really slow compared to `Tuple`, besides, by default all Julia functions should support `size` as an Tuple.

You should at least throw a `depwarn` here using `Base.depwarn` or `@deprecate` for array inputs methods.

Reply:
OK, I will add depwarn. This case is a logical reason for bugging the user.